frontend/components/api/Accounts.js

```diff
@@ -0,0 +1,31 @@
+import { call } from "../../api";
+
+const Accounts = {
+  getAccountInfo: async id => {
+    try {
+      const [rpcAccountResponse, accountResponse] = await Promise.all([
+        call(".nearcore-query", [`account/${id}`, ""]),
+        call(".select", [
+          `SELECT outTransactionsCount.outTransactionsCount, inTransactionsCount.inTransactionsCount FROM
+            (SELECT COUNT(transactions.hash) as outTransactionsCount FROM transactions
+              WHERE signer_id = :id) as outTransactionsCount,
+            (SELECT COUNT(transactions.hash) as inTransactionsCount FROM transactions
+              WHERE receiver_id = :id) as inTransactionsCount
+          `,
+          {
+            id
+          }
+        ])
+      ]);
+
+      console.log(rpcAccountResponse, accountResponse);
```

Comment:
Don't log. If you need logging for testing purposes, check for the right environment or maybe introduce a logger..